js/Overview.md

```diff
@@ -17,17 +17,17 @@ Essential JS is a comprehensive collection of over 80 enterprise-grade HTML5 Jav
 
 * The best way to get started would be to read the "Getting Started" section of the documentation for the component that you would like to start using first. The "Getting Started" guide gives just enough information that you need to know before starting to write code. This is the only section that we recommend reading end-to-end before starting to write code, all other information can be referred as needed.
 
-* Now that you are familiar with the basics of using the component, the next step would be to start integrating the component into your application. A good starting point would be to refer to the code snippets in the [online sample browser](https://js.syncfusion.com/demos/web/) which contains hundreds of code samples, it is very likely that you will find a code sample that resembles your intended usage scenario. 
+* Now that you are familiar with the basics of using the component, the next step would be to start integrating the component into your application. A good starting point would be to refer to the code snippets in the [online sample browser](https://ej2.syncfusion.com/home/) which contains hundreds of code samples, it is very likely that you will find a code sample that resembles your intended usage scenario. 
 
 * After you have integrated the component into your application using one of the sample code snippets as a starting point, it is likely that you would want additional information on specific features and API. The best option would be to search a the specific topic using the search box that is available at the top of the user guide.
 
 * Another valuable resource is the [API reference](https://help.syncfusion.com/api/js/global) which provides detailed information on the object hierarchy as well as the settings available on every object.
 
 ## Additional help resources
 
-* The [Knowledge Base](https://www.syncfusion.com/kb/javascript) section contains responses to some of the most common questions that other customers have asked us in the past so this would be a good place to search for topics that are not covered in the user guide.
+* The [Knowledge Base](https://support.syncfusion.com/kb/retiredproducts/category/94) section contains responses to some of the most common questions that other customers have asked us in the past so this would be a good place to search for topics that are not covered in the user guide.
 
-* Similar to the [Knowledge Base](https://www.syncfusion.com/kb/javascript), the [forum](https://www.syncfusion.com/forums/javascript) section also contains responses to questions that other customers have asked us in the past.
+* Similar to the [Knowledge Base](https://support.syncfusion.com/kb/retiredproducts/category/94), the [forum](https://www.syncfusion.com/forums/javascript) section also contains responses to questions that other customers have asked us in the past.
```

Comment:
your updated URL seems wrong. Please use the below URL, 
https://support.syncfusion.com/kb/web/category/70